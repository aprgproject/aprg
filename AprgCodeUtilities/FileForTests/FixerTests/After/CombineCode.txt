;

// fixPostFixIncrementDecrement
++variable;
--variable;
for(;;++variable1,++variable2,++variable3){}
for(;;--variable1,--variable2,--variable3){}

// fixConstReferenceOrder
Type const & variable;
unsigned int const & variable;
auto const & variable;

Type const variable = nonLiteralValue;
unsigned int const variable = nonLiteralValue;
auto const variable = nonLiteralValue;

const Type variable(nonLiteralValue);
const unsigned int variable(nonLiteralValue);
const auto variable(nonLiteralValue);

// fixNoConstPassByValue
function(unsigned int const variable1, unsigned int const variable2, unsigned int const variable3){}
function(Type const variable1, Type const variable2, Type const variable3){}

//fixConstToConstexpr
constexpr unsigned int variable = 0;
constexpr unsigned int variable=0;

//fixCStylePrintf
cout<<"variable1: ["<< variable1<<"] variable2: ["<< variable2<<"] variable3: ["<< variable3<<"]";
cout<<"";
cout<< i<<"u "<< gsl_vector_get(x, i)<<" "<< gsl_vector_get(xmean, i)<<" "<< gsl_vector_get(xmin, i)<<" "<< gsl_vector_get(xmax, i)<<"\n";
cout<< ti<<" "<< xi<<" "<< medi<<" "<< rmedi<<"\n";

//fixCommentsPositionOfBraces
if {
//Comment
}

// fixCStyleStaticCast
variable1 = static_cast<unsigned int>(variable2;
variable1 + static_cast<unsigned int>(variable2;
