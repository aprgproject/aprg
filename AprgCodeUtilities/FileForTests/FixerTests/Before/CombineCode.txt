;

// fixPostFixIncrementDecrement
++variable;
--variable;
for(;;++variable1,++variable2,++variable3){}
for(;;--variable1,--variable2,--variable3){}

// fixConstReferenceOrder
const Type & variable;
const unsigned int & variable;
const auto & variable;

const Type variable = variable;
const unsigned int variable = variable;
const auto variable = variable;

const Type (variable);
const unsigned int (variable);
const auto (variable);

//fixConstReferenceOrder
unsigned int const variable = variable;
unsigned int const (variable);

// fixCStyleStaticCast
= (unsigned int)variable;
+ (unsigned int)variable;

// fixNoConstPassByValue
(unsigned int variable1, unsigned int variable2, unsigned int variable3)
(Type variable1, Type variable2, Type variable3)

//fixCommentsPositionOfBraces
if //Comment
{
}
