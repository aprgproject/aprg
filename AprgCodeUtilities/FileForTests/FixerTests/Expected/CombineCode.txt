;

// fixPostFixIncrementDecrement
++variable;
--variable;
for(;;++variable1,++variable2,++variable3){}
for(;;--variable1,--variable2,--variable3){}

// fixConstReferenceOrder
Type const & variable;
unsigned int const & variable;
auto const & variable;

Type const variable = variable;
unsigned int const variable = variable;
auto const variable = variable;

Type const (variable);
unsigned int const (variable);
auto const (variable);

//fixConstToConstexpr
constexpr unsigned int variable = 0;
constexpr unsigned int variable=0;

// fixCStyleStaticCast
= static_cast<unsigned int>(variable;
+ static_cast<unsigned int>(variable;

// fixNoConstPassByValue
(unsigned int const variable1, unsigned int const variable2, unsigned int const variable3)
(Type const variable1, Type const variable2, Type const variable3)

//fixCommentsPositionOfBraces
if {
//Comment
}
